src/css/styles.scss

```diff
@@ -1,7 +1,18 @@
+$dark-lime-green: #359567;
```

Comment:
Just for the sake of future-proofing our variable names, instead of listing what the color is NOW should we maybe list it by the abstract of what role the color is playing? Like `primaryColor' or 'CardBorder' or 'accent2' etc.

Reply:
Love this idea. I will make that change at another time after reviewing what each color scheme is used for.

Reply:
Update: I made this a low priority issue on the SASS milestone so that it isn't forgotten. 
Addresses #52